docs/root/start/sandboxes/postgres.rst

```diff
@@ -0,0 +1,134 @@
+.. _install_sandboxes_postgres:
+
+Postgres Filter
+===============
+
+In this example, we show how the :ref:`Postgres filter <config_network_filters_postgres_proxy>` 
+can be used with the Envoy proxy. The Envoy proxy configuration includes a Postgres filter that
+parses queries and collects Postgres-specific metrics.
+
+
+Running the Sandboxes
+~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: _include/docker-env-setup.rst
+
+
+Step 3: Build the sandbox
+*************************
+
+.. code-block:: console
+
+  $ pwd
+  envoy/examples/postgres
+  $ docker-compose pull
+  $ docker-compose up --build -d
+  $ docker-compose ps
+
+         Name                      Command             State                             Ports                          
+  ----------------------------------------------------------------------------------------------------------------------
+  postgres_postgres_1   docker-entrypoint.sh postgres  Up      0.0.0.0:5432->5432/tcp                                   
+  postgres_proxy_1      /docker-entrypoint.sh /usr ... Up      10000/tcp, 0.0.0.0:1999->1999/tcp, 0.0.0.0:8001->8001/tcp
+
+
+Step 4: Issue commands using psql
+*********************************
+
+Use ``psql`` to issue some commands and verify they are routed via Envoy. Note
```

Comment:
s/Use psql/This example uses a psql client inside a container/

Reply:
Fixed!